src/Client/wwwroot/data/corpses.json

```
@@ -49,6 +49,14 @@
       "description": "an unreleased survival game that was in development by Blizzard and cancelled after the acquisition of Activision Blizzard by Microsoft",
       "link": "https://www.msn.com/en-us/entertainment/gaming/blizzard-cancels-survival-game-following-layoffs/ar-BB1hftuk"
     },
+    {
+      "name": "Dynamics 365 Human Resources (Stand alone)",
```

Just a note, the parenthetical is redundant in the name. The qualifier will generate a parenthetical for us if not null.

I expect this will render as "Dynamics 365 Human Resources (Stand alone) (Stand-alone)" as is.  If you would like to fix this yourself, that'd be appreciated.

```suggestion
      "name": "Dynamics 365 Human Resources",
```

---

@victorfrye Thanks for suggestion, committed.